db/config.cc

```diff
@@ -871,6 +871,9 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , enable_sstables_mc_format(this, "enable_sstables_mc_format", value_status::Unused, true, "Enable SSTables 'mc' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , enable_sstables_md_format(this, "enable_sstables_md_format", value_status::Unused, true, "Enable SSTables 'md' format to be used as the default file format.  Deprecated, please use \"sstable_format\" instead.")
     , sstable_format(this, "sstable_format", value_status::Used, "me", "Default sstable file format", {"md", "me"})
+    , uuid_sstable_identifiers_enabled(this,
+            "uuid_sstable_identifiers_enabled", value_status::Used, true, "If set to true, each newly created sstable will have a UUID "
```

Comment:
If it's possible to make it LiveUpdate easily (e.g. no notifers etc) please make it so.

It's not critical: even if it's live updateable, actual sstables on disk will have to be rewritten if the user doesn't want uuid. So LiveUpdate solves only a small part of the downgrade effort. But if it's easy then we should do it.